.gitignore

```diff
@@ -4,3 +4,5 @@ coverage
 tests/runtime
 coverage.xml
 .vagrant
+.idea
+.DS_STORE
```

Comment:
You too :) what i must do next, that you added my pull request? ) rewrite .gitignore?